xarray/tests/test_formatting.py

```diff
@@ -88,27 +109,27 @@ def test_format_items(self):
 
     def test_format_array_flat(self):
         actual = formatting.format_array_flat(np.arange(100), 13)
-        expected = '0 1 2 3 4 ...'
+        expected = '0 1 ... 98 99'
         assert expected == actual
 
         actual = formatting.format_array_flat(np.arange(100.0), 11)
-        expected = '0.0 1.0 ...'
+        expected = '0.0 ... 99.0'
         assert expected == actual
 
         actual = formatting.format_array_flat(np.arange(100.0), 1)
-        expected = '0.0 ...'
+        expected = '0.0 ... 99.0'
         assert expected == actual
 
         actual = formatting.format_array_flat(np.arange(3), 5)
         expected = '0 1 2'
         assert expected == actual
 
         actual = formatting.format_array_flat(np.arange(4.0), 11)
-        expected = '0.0 1.0 ...'
+        expected = '0.0 ... 3.0'
         assert expected == actual
 
         actual = formatting.format_array_flat(np.arange(4), 0)
-        expected = '0 ...'
+        expected = '0 ... 3'
         assert expected == actual
 
```

Comment:
Can you add tests that pull out different numbers of start/end items, e.g., expected output like `0 1 ... 99` or `0 1 2 ... 98 99`?